website/source/layouts/docs.erb

```diff
@@ -326,6 +326,24 @@
           <li<%= sidebar_current("docs-commands-monitor") %>>
             <a href="/docs/commands/monitor.html">monitor</a>
           </li>
+          
+          <li<%=sidebar_current("docs-commands-namespace") %>>
+            <a href="/docs/commands/namespace.html">namespace</a>
+            <ul class="nav">
+              <li<%= sidebar_current("docs-commands-namespace-delete") %>>
```

Comment:
Is there a reason, why you put delete first? Having list and read in the front feels more natural for me.

Reply:
Our other sidebar entries are alphabetical so I stuck with that for consistencies sake.